src/elements/OcCheckbox.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  <input
-    class="oc-checkbox"
-    type="checkbox"
-    v-model="checkboxState"
-    @change="$_ocCheckbox_change($event)"
-    @click.stop
-  />
+  <label>
+    <input
+      class="oc-checkbox"
+      type="checkbox"
+      v-model="checkboxState"
+      @change="$_ocCheckbox_change($event)"
+      @click.stop
+    />
+    <span v-text="label" :class="labelVisuallyHidden ? 'oc-visually-hidden' : ''" />
```

Comment:
This way you can drop the ternary operator.

```suggestion
    <span v-text="label" :class="{ 'oc-visually-hidden': labelVisuallyHidden }'" />
```

Reply:
Cheers. Edited.